include/mega/megaclient.h

```diff
@@ -277,6 +277,9 @@ class MEGA_API MegaClient
     // retrieve user details
     void getaccountdetails(AccountDetails*, bool, bool, bool, bool, bool, bool);
 
+    // check if the available bandwidth quota is enough to transfer an amount of bytes
+    void querybandwidthquota(m_off_t size);
+
```

Comment:
Maybe we should refactorize all these to be _transfer quota_, which is the actual name of the quota and how it's displayed to users.